.github/workflows/test-harness-verity-afj.yml

```diff
@@ -0,0 +1,53 @@
+name: test-harness-verity-afj
+# RUNSET_NAME: "Verity to Javascript"
+# Scope: AIP 1.0
+# Exceptions: None
+#
+# Summary
+#
+# This runset uses the current main branch of Javascript for all of the agents except Acme (Issuer),
+# which uses the dev branch of Evernym Verity. The runset covers all of the AIP 1.0 tests
+# under the connection protocol RFC1060.
+#
+# Current
+#
+# All the Connection tests are running. All tests are passing
```

Comment:
Best to revise this. "Initial implementation to get the tests running smoothly. Currently testing connection only."